### DISCUSSION TITLE
Using the same trait-type name multiple times in each image

### DISCUSSION THREAD

**Discussion by jjclay** [nftchef/art-engine#153] [Help + Q&A]
Hello,
I've tried every scenario I can think of and am not sure if this is possible.  I would like to have multiple trait-type: "accessory" on one image, like this:
![image](https://user-images.githubusercontent.com/67430365/164950323-d041c882-9aa3-4fde-b586-d2a266898873.png)
I know that if I set useRootTraitType to true I can use the root directory "accessory" for different types of accessories all located in sub-directories under "accessory".  The problem is, if the sub-directories are not weighted (i.e. #100.png), the root directory name is not used and I'm back to having separate trait-types for each accessory.  If they are weighted, then only one trait will be picked from all of the sub-directories.  I would like at least 1 pick from each of the sub-directories, giving the image multiple accessories as in the attached example.  Is it possible to do what I'm asking?
Thank you!

**Comment by BakeSloth** [comment]
Why not just duplicate the layer 4 times?
in config.js

    layersOrder: [
      { name: "Background"},
      { name: "Body"},
      { name: "Outfit"},
      { name: "Eyes"},
      { name: "accessory"},
      { name: "accessory"},
      { name: "accessory"},
      { name: "accessory"},
    ],
